pkg/ibmvpc/analysis_output_test.go

```diff
@@ -68,6 +68,8 @@ type vpcGeneralTest struct {
 	EDstMinPort    int64
 	EDstMaxPort    int64
 	detailExplain  bool
+	enable         []string
+	disable        []string
```

Comment:
It would make much more sense if each subcommand will extend a basic Testcase struct with its own required fields.
Currently this struct is a mess, as it serves report, diff, explain and lint

Reply:
Lets have it in a separate PR.
Some of the struct and functionality is common, and doing this separation properly will require some software engineering work  which is not related to this PR.

Reply:
OK. Please open an issue

Reply:
https://github.com/np-guard/vpc-network-config-analyzer/issues/765
Please connect to a milestone